### ISSUE TITLE
Command Line Basics: WSL2 instructions not clear enough

### ISSUE THREAD

**Issue by scheals** [TheOdinProject/curriculum#27338]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

There is a note for WSL2 users before the assignment:

> Note for WSL2 users: You will have to use the wget command along with the link given in the Download files section in order to have the zip file in your WSL2 installation (wget https://swcarpentry.github.io/shell-novice/data/shell-lesson-data.zip). You will also have to install unzip by using the command sudo apt install unzip and then unzip shell-lesson-data.zip to unzip the file. **Keep in mind that throughout the course linked in the first step below, your terminal output may look slightly different to what is shown in the lessons. Anytime the course asks you to go to the Desktop, you will instead be going to the home directory which can be done by using the cd command (cd ~).**

Slightly different, eh? WSL2 has nothing in its home directory. Why is that a problem? Let's see [the first episode of the resource](https://swcarpentry.github.io/shell-novice/02-filedir.html) that uses any commands:
![image](https://github.com/TheOdinProject/curriculum/assets/95084627/f551b3cf-3d5b-4dc8-b56b-e6806dfa28b5)

What will be the output for a WSL2 user? If they remembered about the note and ran the `wget` and the `unzip`, they'll at least see `shell-lesson-data.zip` and `shell-lesson-data` directory. If they did not, they will see absolutely nothing.

Now another curveball that probably should be self-explanatory due to taking into account potential differences but nonetheless is a pain point for our learners:
![image](https://github.com/TheOdinProject/curriculum/assets/95084627/80147e74-e7d5-4953-8576-2231474901ce)

The note is exactly referring to a case like this. What command should they run instead? The problem only compounds itself with next commands used:
```bash
ls -F Desktop/shell-lesson-data

# comes next
cd Desktop
cd shell-lesson-data
cd exercise-data
```
The easiest way to help with this is most likely adding something like this to the note:
> Your file structure will look wildly differently from the one exemplified in the resource - WSL2 has no directories at all in its home directory. So it is easier for you to follow, we'll emulate just a small, relevant part of it. Run these commands in order:
```bash
mkdir Desktop
cd Desktop
```
and changing the rest of the note accordingly. Will this help? Maybe somewhat. The main issues are somewhere else but I'm jaded so I'll keep them to myself.

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-command-line-basics

**Comment by CouchofTomato** [comment]
I wonder if we can find a better resource for WSL2 users?

**Comment by thatblindgeye** [comment]
@scheals just to be clear, you're suggesting we update the note for WLS2 users to instruct them to first create and `cd` into a Desktop directory, and then download/extract the lesson data zip file in that Desktop directory?

If this would only maybe help somewhat then I'd question how valuable this change is. If there is a larger issue it'd be worth exploring that instead.

**Comment by scheals** [comment]
Yes, I am suggesting that so WSL2 users have the same directory structure as the other options. The current note must work for some people but many forget about it the moment they go to the linked resource. Making them create the Desktop directory makes it so it doesn't matter if they forget. That'd be probably enough.

It is probably one of the top 3 pain points with WSL2, the others being issues with opening `.html` files and various issues stemming from WSL2/Windows integration (using Windows file system, using Windows installed Node and many more).

The larger issue is probably the jump from using software with GUI into CLI *and* that software being very strongly integrated with Windows. I believe two past changes in the installation lessons got rid of most people who went for WSL2 because they thought it is avoiding Linux and that cohort had the hardest time reasoning about WSL2.

**Comment by bensterne** [comment]
Somewhat related to this issue but slightly different is the data download of the zip at that website is not working for me.

**Comment by xandora** [comment]
It's not labelled (Advanced) for no reason... 😅 

There's a balance between spoon feeding and expecting someone picking the option marked Advanced to know what that means that we're not quite getting right, I guess.

Given the number of times I've seen someone simply copy and paste the code block examples without reading the context between, I don't think we'll ever completely stop questions about this particular topic.

Someone probably needs to stare at the WSL instructions and wait for some divine inspiration before we figure this one out.

**Comment by Asartea** [comment]
my two cents is that in these specific circumstances, it might be worth adding a shout-out because the article is instructing them to do something that isn't possible, and I'm willing to go well maybe people just picked it because its the most Windowsy option. I do agree in general terms that Advanced things are allowed to be advanced

**Comment by KevinMulhern** [comment]
![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExZWZlczA4a3lkdjZyNmF6cnNhc3g1Z3V1Y3RvODhiN2M0Zzl1dDl2NiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/YA6dmVW0gfIw8/giphy.gif)

**Comment by Nayab-Gauhar** [comment]
Hi! I'd like to work on this issue for Hacktoberfest. Based on the thorough discussion here, I can help clarify the WSL2 instructions in the Command Line Basics lesson by:

• Implementing @scheals's suggestion to add a "Desktop emulation" step in the WSL2 note (mkdir Desktop && cd Desktop)
• Updating the note to explain why WSL2 home directory looks different (empty vs. the resource's screenshots)
• Clarifying that WSL2 users should replace "cd Desktop" commands with "cd ~/Desktop" or just staying in ~/Desktop
• Adding a note about file structure differences upfront to reduce confusion
• Testing the updated instructions to ensure they work smoothly with the linked Software Carpentry resource

I'll update the markdown file to bridge the gap between WSL2's actual state and the external resource's assumptions. Could you please assign this issue to me? Thanks!

**Comment by JoshDevHub** [comment]
This was closed by #28127 it seems.

If there's something I'm not catching about it, feel free to comment here for further discussions.
